css/activity.css

```diff
@@ -5,3 +5,7 @@
 #main-toolbar #new-button {
   background-image: url(../icons/new-button.svg);
 }
+
+#main-toolbar #clear-button {
+  background-image: url(../icons/clear-button.svg);
+}
```

Comment:
You can remove the No new line at EOF by running `echo >> css/activity.css`, this makes it better to view when using diff tools or cat.

Reply:
done...